projects/components/src/overlay/overlay.service.ts

```diff
@@ -8,8 +8,12 @@ import { ModalOverlayConfig } from './modal/modal';
 import { ModalOverlayComponent } from './modal/modal-overlay.component';
 import { SheetOverlayComponent } from './sheet/sheet-overlay.component';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
```

Comment:
If providing this in root, the default injector should be removed - it needs to be provided.

Reply:
Looked into this a bit, I think it's OK to default it after all. My understanding _was_ that because these are used to resolve a component factory, only components available in the root module could be instantiated. It looks like that's not the case (maybe it changed with ivy, maybe it never was). I suspect it does still hold with providers, but we use view/local providers less frequently, so we can get away with defaulting I think.

Reply:
okay. Thanks for explaining!